cloudbuild.yaml

```diff
@@ -12,17 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Builds docker image for gcp-variant-transforms:
-# Run using:
-# $ gcloud container builds submit --config cloudbuild.yaml --timeout 1h .
-substitutions:
-  _CUSTOM_TAG_NAME: 'latest'
+# This is used by Google Cloud Build GitHub app, which automatically builds the
+# code each time you push code to Github. This is what it does:
```

Comment:
This is actually post commit (eg, after pushed to master) or is it when a PR is created?

Reply:
It is supposed to work when we create a PR, but the automatic running of CloudBuild is not supported for the forked repository yet.